README.md

```diff
@@ -1,248 +1,236 @@
 ```scss
 .o-example {
-	content: oBrandGet('o-example', 'component-content'); // default value
+	background: oBrandGet($component: 'o-example', $variables: 'example-background'); // background: paper;
 }
 ```
 
-### Retrieve A Variable For A Variant
-
-To retrieve a variable for a variant use `oBrandConfigureFor`. This mixin accepts a content block and variant. This indicates styles within the content block should pertain to the passed variant.
-
-Styles within `oBrandConfigureFor` are [only output if the current brand supports the variant](#output-styles-only-if-a-brand-supports-a-variant).
-
-Building on the "define" example above:
+To request multiple variables at once, assuming variables `example-border-size: 1px`, `example-border-type: solid`, `example-border-color: grey` are defined using `oBrandDefine`:
 ```scss
-@include oBrandConfigureFor('o-example', 'inverse') {
-	.o-example--inverse {
-		content: oBrandGet('o-example', 'component-content'); // "inverse" variant value
-	}
+.o-example {
+	border: oBrandGet($component: 'o-example', $variables: ('example-border-size', 'example-border-type', 'example-border-color')); // border: 1px solid grey;
 }
 ```
 
-To retrieve a variable for a compound variant pass a list:
+Retrieve a variable for a variant using `oBrandGet` using `$force-variant`. Whilst this is possible use [`oBrandConfigureFor`](#retrieve-a-variable-for-a-variant) instead where possible.
 ```scss
-@include oBrandConfigureFor('o-example', ('inverse', 'demo')) {
-	.o-example--inverse.o-example--demo {
-		content: oBrandGet('o-example', 'component-content'); // "inverse demo" variant value
-	}
+.o-example--inverse {
+	background: oBrandGet($component: 'o-example', $variables: 'example-background', $force-variant: 'inverse'); // background: slate;
 }
 ```
 
-Nesting is also supported:
-```scss
-@include oBrandConfigureFor('o-example', 'inverse') {
-	@include oBrandConfigureFor('o-example', 'demo') {
-		.o-example--inverse.o-example--demo {
-			content: oBrandGet('o-example', 'component-content'); // "inverse demo" variant value
-		}
-	}
-}
-```
+- `oBrandGet` returns `null` if a variable is undefined. Sass removes css properties which are set to `null`. This is a useful feature to conditionally output css properties for different variants.
 
-A variable may also be retrieved for an explicit variant using the `$force-variant` argument of `oBrandGet`. This is useful when retrieving a variant value within a function, or when a specific value is needed regardless of what variant is configured by `oBrandConfigureFor`.
+### oBrandConfigureFor
 
-```scss
-	.o-example--inverse {
-		content: oBrandGet('o-example', 'component-content', $force-variant: 'inverse'); // "inverse" variant value
-	}
-```
+`oBrandConfigureFor` improves working with variants. It accepts a Sass content block which will only output if the brand supports the given variant. It also configures all `oBrandGet` calls within its content block for a given variant.
```

Comment:
I'm a little confused by this sentence... what is it improving? And what will it output? Do you mean something like ('it [sass content block or oBrandConfigureFor?] will only return an output if the brand supports the given variant' ?